server.js

```diff
@@ -12,7 +12,10 @@ var eventEmitter = require('events').EventEmitter
 var hostname = process.env.MULT_WEB_SEQ_SERV || 'localhost';
 var base_path = process.env.BASE_PATH || '';
 var hostnamePort = process.env.MULT_WEB_SEQ_SERV_P || '8080';
+var adminPassword = process.env.ADMIN_PASSWORD || 'admin';
```

Comment:
I think we should avoid to define a default admin password.

Reply:
This is just for development.
In production you should make sure you add the environment variable.